src/game/tests/testGrid.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+from ..start import init_grid, creat_grid_string
+
+
+class TestGridMethods(unittest.TestCase):
+
+    def test_height_4(self):
```

Comment:
You test names aren't really meaningful.

Please describe the expected output of your tests, like:
`test_init_grid_should_render_the_correct_amount_of_columns`
`test_init_grid_should_render_the_correct_amount_of_rows`
`test_init_grid_should_fill_a_matrix_with_dots`

And so on